src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1065,6 +1058,9 @@ internal void ComputeFrameUpdate()
         private void OnSceneItemsChanged()
         {
             UpdateSceneItems();
+            //TODO unclear if these are still required.
+            //since sceneItems are observable and we force them to update
+            //by clearing and readdding all items.
             RaisePropertyChanged("SceneItems");
```

Comment:
I dont see any consumer of this property change even before, and the other two places are both commented out.. Making me thinking if we can totally remove this

Reply:
well the consumer is implicit, it's the xaml binding.